src/mainwindow.cpp

```diff
@@ -33,15 +33,16 @@
 #include <QSortFilterProxyModel>
 
 #include "models/costmodel.h"
-#include "models/framedata.h"
 #include "parsers/perf/perfparser.h"
 #include "flamegraph.h"
+#include "models/summarydata.h"
 
 MainWindow::MainWindow(QWidget *parent) :
     QMainWindow(parent),
     ui(new Ui::MainWindow),
     m_bottomUpCostModel(new CostModel(this)),
-    m_parser(new PerfParser(this))
+    m_parser(new PerfParser(this)),
+    m_topHotspotsCostModel(new CostModel(this))
```

Comment:
this is not needed, the top 'N' entries uses the same data, and thus the same model, as the bottomUpCostModel. only a proxy is missing (cf. topproxy in heaptrack)